README.md

```diff
@@ -208,6 +208,7 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-99-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-98-orange.svg?style=flat-square)](#contributors-)
```

Comment:
This is duplicate with line 210.
Please remove one, then run the `npm run contributors:generate` command in your terminal.